src/rail/estimation/algos/deepdisc.py

```diff
@@ -165,9 +165,6 @@ def inform(self, input_data, input_metadata):
             self.finalize()
         return self.get_handle("model")
 
-    def finalize(self):
```

Comment:
Turns out that it's super important to _not_ overwrite `finalize` because that is the function that will go around and remove all the "inprogress_*" prefixes from the various output files.